pygeoapi/config.py

```diff
@@ -40,6 +40,14 @@
 THISDIR = Path(__file__).parent.resolve()
 
 
+def load_json_schema() -> dict:
```

Comment:
`s/load_json_schema/load_schema`?

Reply:
@tomkralidis Not sure what you mean by this?

I just put the code responsible for the actual schema loading out of `validate_config` and into its own function so I can reuse it (i.e. in the tests).

Reply:
@GeoSander I guess he's asking to change the function name to just `load_schema`

Reply:
Understood :)
Will rename it!